app/src/main/java/com/internship/move/network/ServiceApi.kt

```diff
@@ -0,0 +1,27 @@
+package com.internship.move.network
+
+import com.internship.move.network.model.AddLicenseResponse
+import com.internship.move.network.model.LoginRequest
+import com.internship.move.network.model.LoginResponse
+import com.internship.move.network.model.RegisterRequest
+import com.internship.move.network.model.RegisterResponse
+import okhttp3.MultipartBody
+import okhttp3.RequestBody
+import retrofit2.http.Body
+import retrofit2.http.Multipart
+import retrofit2.http.POST
+import retrofit2.http.PUT
+import retrofit2.http.Part
+
+interface ServiceApi {
+
+    @POST("users/register")
+    suspend fun register(@Body registerRequest: RegisterRequest): RegisterResponse
```

Comment:
Personally I prefer adding the `Dto` (or `DTO`) suffix to API classes. It stands for "Data Transfer Object" and it makes it clearer what these classes are for, and it's easier to search for them